push/model/src/main/java/org/jboss/aerogear/unifiedpush/message/Criteria.java

```diff
@@ -82,6 +83,37 @@ public void setVariants(List<String> variants) {
         this.variants = variants;
     }
 
+    /**
+     * Helper method to check if only categories are applied. Usful in GCM land, where we use topics
+     */
+    @JsonIgnore
+    public boolean isCategoryOnly() {
```

Comment:
Perhaps these methods should be moved to a util, next to TokenLoader?

Not sure if this is generally helpful on the `Criteria` API class